network/network.test.js

```diff
@@ -113,6 +113,19 @@ describe('useNetworkStatus', () => {
     expect(result.current.effectiveConnectionType).toEqual('4g');
   });
 
+  test('should update the effectiveConnectionType state when network get disabled', () => {
```

Comment:
It's probably worth explicitly testing defining `navigator.onLine` with `true/false` states prior to calling `renderHook` as well.